DQMOffline/Trigger/plugins/HLTTagAndProbeOfflineSource.cc

```diff
@@ -51,16 +38,10 @@ template <typename TagType, typename TagCollType, typename ProbeType, typename P
 void HLTTagAndProbeOfflineSource<TagType, TagCollType, ProbeType, ProbeCollType>::fillDescriptions(
     edm::ConfigurationDescriptions& descriptions) {
   edm::ParameterSetDescription desc;
-  desc.add<edm::InputTag>("objs", edm::InputTag(""));
   desc.addVPSet("tagAndProbeCollections",
                 HLTDQMTagAndProbeEff<TagType, TagCollType, ProbeType, ProbeCollType>::makePSetDescription(),
-                std::vector<edm::ParameterSet>());
-
-  // addDefault must be used here instead of add unless this function is specialized
-  // for different sets of template parameter types. Each specialization would need
-  // a different module label. Otherwise the generated cfi filenames will conflict
-  // for the different plugins.
-  descriptions.addDefault(desc);
+                {});
+  descriptions.addWithDefaultLabel(desc);
```

Comment:
This change is unrelated to the main goal of this PR. It correctly creates the following `cfi` files (one per template instantiation):
```bash
"${CMSSW_BASE}"/cfipython/"${SCRAM_ARCH}"/DQMOffline/Trigger/hltElePhoTagAndProbeOfflineSource_cfi.py
"${CMSSW_BASE}"/cfipython/"${SCRAM_ARCH}"/DQMOffline/Trigger/hltEleTagAndProbeOfflineSource_cfi.py
"${CMSSW_BASE}"/cfipython/"${SCRAM_ARCH}"/DQMOffline/Trigger/hltMuEleTagAndProbeOfflineSource_cfi.py
"${CMSSW_BASE}"/cfipython/"${SCRAM_ARCH}"/DQMOffline/Trigger/hltMuPhoTagAndProbeOfflineSource_cfi.py
"${CMSSW_BASE}"/cfipython/"${SCRAM_ARCH}"/DQMOffline/Trigger/hltMuTagAndProbeOfflineSource_cfi.py
"${CMSSW_BASE}"/cfipython/"${SCRAM_ARCH}"/DQMOffline/Trigger/hltPhoTagAndProbeOfflineSource_cfi.py
```